lib/improver/tests/spotdata/spotdata/test_NeighbourSelection.py

```diff
@@ -660,6 +664,28 @@ def test_global_nearest(self):
         self.assertIsInstance(result, iris.cube.Cube)
         self.assertArrayEqual(result.data, expected)
 
+    def test_global_returned_site_coordinates(self):
+        """Test that the site coordinates in the returned neighbour cube are
+        latitudes and longitudes. Here the input site list contains site
+        coordinates defined in latitudes and longitudes, so they should be
+        unchanged."""
+
+        plugin = NeighbourSelection()
+        result = plugin.process(self.global_sites, self.global_orography,
```

Comment:
To ensure that the plugin doesn't modify its arguments:
```suggestion
        result = plugin.process(self.global_sites.copy(), self.global_orography,
```

Reply:
I have just moved the setting of the expected results to above the call to the plugin.

Reply:
I'm happy now that the result and expected values have different memory addresses.